pipeline/recon/web/subdomain_takeover.py

```diff
@@ -120,9 +121,9 @@ def run(self):
             return
 
         command = [
-            tool_paths.get("tko-subs"),
+            tools.get("tko-subs").get("tool_path"),
             f"-domain={','.join(domains)}",
-            f"-data={tool_paths.get('tko-subs-dir')}/providers-data.csv",
+            f"-data={tools.get('tko-subs').get('git_dir')}/providers-data.csv",
```

Comment:
after seeing how many changes it would take to make this uniform (if hyphens were used), ill leave this up to you whether you want to do it or not.  this is the only location in code you see it, the rest is in the configs or the `.get` is called on a single word `fingerprints` et al